src/domain/community/community/community.service.ts

```diff
@@ -257,6 +258,20 @@ export class CommunityService {
     });
   }
 
+  async countUsersWithRole(
```

Comment:
You have ```CommunityContributorType``` enum, why not make a countContributorPerRole method and have the enum as input. The only difference between countUsersWithRole and countOrganizationsWithRole is one call, that can be determined by the enum - all the rest is the same (signature, return type, the membership credential call etc.).